development/README.md

```diff
@@ -290,7 +290,29 @@ $BUILD_SCRIPTS/qgis-dev-install.sh $BUILD_DIR
 * Make a directory in your QGIS source tree called ``build``
 * Open the top level ``CMakeLists.txt`` in QtCreator
 
-### Making a kit
+### Set up a ccache compile profile in QtCreator
+
+CCache will cache previously compiled objects and greatly speed up the compilation process. We will make an alias for compiling c++ sources. Make sure you did ``brew install ccache`` when you were installing your dependencies further up in this document:
```

Comment:
I'd strip the "when you were..." part. After all, if you haven't done it back then, you should do it now.